### ISSUE TITLE
Page landmarks should be tagged and described for accessibility technology

### ISSUE THREAD

**Issue by joychen17** [joychen17/joychen17.github.io#2]
Landmarks such as main content and footer should be built using their corresponding HTML5 element or include a description using an "aria-label" attribute and a "role" tag that equals to "contentinfo" or "main". Landmarks such as search and navigation are tested in other sections.

Failed Elements

<div class="main_content"> <h1> Joy Chen </h1>  <p> Welcome to my personal webpage....
